README.md

```diff
@@ -103,6 +103,9 @@ a few things you can personalize:
     When changing the default, resident credentials become undiscoverable without
     user verification. This helps privacy, but can make usage less comfortable
     for credentials that need less protection.
+6.  Increase the default minimum length for PINs in `ctap/storage.rs`.
+    The current minimum is 4. Values from 4 to 63 are allowed.
```

Comment:
Maybe you can add a rationale as to why one would want to increase this default minimum? (I guess to avoid users setting too easy PINs?)

Reply:
Provided.